plugin/clang_complete.vim

```diff
@@ -477,7 +477,13 @@ endfunction
 
 function! s:TriggerSnippet()
   " Restore original return key mapping
-  silent! execute 'inoremap <script> <buffer> <silent> <CR> '.s:old_cr
+  silent! execute s:old_cr.mode.
+      \ (s:old_cr.noremap ? 'noremap '  : 'map').
```

Comment:
The 'i' comes from `s:old_cr.mode`. That makes the code reusable later on for non-insert mode maps.